### ISSUE TITLE
Recoverable Errors

### ISSUE THREAD

**Issue by jonnio** [BerryWorksSoftware/edi-json#7]
The XML parser allows for the use of an [EDISyntaxExceptionHandler](https://github.com/BerryWorksSoftware/edireader/blob/master/edireader/src/main/java/com/berryworks/edireader/error/EDISyntaxExceptionHandler.java) to ignore errors and move on anyway. This feature should also be provided for JSON data when creating the edi json parser.

**Comment by canabrook** [comment]
Yes, I agree. I will make sure that is include in the next release.

**Comment by bolognam** [comment]
I would also like this feature to be implemented, maybe using a command-line flag of "-r" just like the edi-to-xml parser you wrote.
